src/clients/c/test.zig

```diff
@@ -244,6 +244,10 @@ test "c_client tb_status" {
     // All other status are not testable.
 }
 
+test {
+    _ = @import("tb_client/signal.zig");
```

Comment:
Include the at the top-level, in unit_tests.zig. I don't have a strong argument for one list versus
hierarchical tree of inclusions, but one list is what we do today!